lib/views/post/widgets/post_vote.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:pr0gramm/api/dtos/item/item.dart';
+import 'package:pr0gramm/entities/enums/vote.dart';
+import 'package:pr0gramm/entities/enums/vote_button_type.dart';
+import 'package:pr0gramm/services/vote_animation_service.dart';
+import 'package:pr0gramm/services/vote_service.dart';
+import 'package:pr0gramm/views/post/widgets/vote_button.dart';
+import 'package:pr0gramm/widgets/global_inherited.dart';
+
+class PostVote extends StatelessWidget {
+  final Item item;
+  final Vote initialVote;
+  final VoteService _voteService = VoteService.instance;
+
+  PostVote({
+    Key key,
+    this.item,
```

Comment:
Add the @required tag for all required parameters.

Reply:
fixed with edf2f57